### ISSUE TITLE
command line curv tool does not respect silent

### ISSUE THREAD

**Issue by dcanelhas** [CloudCompare/CloudCompare#750]
## Current Behavior
A dialog pops up to show progress (as normally would happen when -SILENT is not specified)

## Expected Behavior
feedback through stdout if any.

## Steps to Reproduce (for bugs)
```bash
$ CloudCompare -SILENT -O points.ply -CURV MEAN 0.5
[LOADING]
Opening file: '038.ply'
Found one cloud with 12933577 points
[CURVATURE]
	Kernel size: 0.5
```
at this point the octree progress bar shows up and the filter begins to run.

## Context/Comments
There is probably some conditional statement missing, since the silent flag works fine for e.g. -OCTREE_NORMALS 0.5

## Your Environment
* CC Version: 2.10-alpha
* OS & Version: Arch Linux x86_64 (4.16.12-1-ARCH Fri May 25)
* Graphics card: n.a.

**Comment by dgirardeau** [comment]
Indeed, propagating this 'silent' flag in all the functions of CC is a nightmare ;)

**Comment by dgirardeau** [comment]
Okay this is fixed: 043be476e2c5ed9ba12663e9d871ba1c59a7464f.

Thanks for the feedback.
